TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/Screens/QuickSetupCompleteScreen.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using UnityEngine;
+using Ultraleap.TouchFree.ServiceShared;
```

Comment:
This doesn't appear to be used, is it necessary?

Reply:
It isn't, I think this is carry-over from one of the other commits